content/en/blog/low-hanging-performance/index.md

```diff
@@ -0,0 +1,460 @@
+---
+date: 2023-09-04
+title: "How To Write Readable & Performant Go Code"
+linkTitle: "Performant Go Code"
+description: "The Go programming language has an excellent [concurrency 
+model](https://findy-network.github.io/blog/2023/06/22/beautiful-state-machines-fsm-part-ii/)
+that offers great potential to utilize the power of multi-core CPUs. However, we
+need to understand the basics of the single CPU core for the overall software
+performance. Common programming wisdom says you shouldn't prioritize performance
+over other [software architecture quality
+attributes](https://en.wikipedia.org/wiki/List_of_system_quality_attributes)
+during programming. They say you shouldn't sacrifice, for instance, readability
+or maintainability for performance. You can write more readable code when you
+know how compilers and underlying hardware work and execute **multi-discipline
```

Comment:
what are these multi-disclipline engineering practices? is it explained earlier or in some of the chapters?